omod/src/main/webapp/fragments/field/codedPersonAttributes.gsp

```diff
@@ -0,0 +1,10 @@
+<p>
+  <select name="" size="6">
+    <option value ="LFHC:901">Lao</option>
+    <option value ="LFHC:902">Hmong</option>
+    <option value ="LFHC:905">Khmuic</option>
+    <option value ="LFHC:906">Other Lao</option>
+    <option value ="LFHC:911">Foreigner</option>
+  </select>
+  <span class="field-error"></span>
+</p>
```

Comment:
I'm confused... this seems like a set of implementation-specific attributes?

Reply:
@mogoodrich my understanding of a person coded attribute as per the ticket was to come up with a specific attribute and display it using a drop down!! could i be missing out some thing??

Reply:
@HerbertYiga I'm also confused.  Have a look at RA-982 and read through the ticket history.  One thing to note is that a pull request is already outstanding for this work and needs someone to take it over and resolve comments on it.  Have a look at that pull request - it might help you understand the nature of the solution that is needed.  Thanks!